client/src/util/stateManager/schemaHelpers.ts

```diff
@@ -5,8 +5,14 @@
  * like immutable.js
  */
 
+import cloneDeep from "lodash.clonedeep";
```

Comment:
Schema helpers gained utilities to add/remove writable columns, which the autosave thunks use when reconciling local changes with persisted data.